### PR TITLE
zh-translation: docs/reference/config/analysis/ist0117/index.md

### PR DESCRIPTION
**Pull request by MXuDong**
[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

File: content/zh/docs/reference/config/analysis/ist0117/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+layout: analysis-message
+owner: istio/wg-user-experience-maintainers
+test: no
+---
+
+当 Pods 资源没有与任何 Service 资源相关联时会出现该问题。
+
+一个 Pod 即使没有暴露任何端口也必须被至少一个 Kubernetes 的 Service 资源关联。
+
+请查看 [Istio 要求](/zh/docs/ops/deployment/requirements/index.md)来了解更多信息。
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/analysis/ist0117/index.md:12]: 没有 index.md 哦。

**Review by kebe7jun — CHANGES_REQUESTED** [review]

File: content/zh/docs/reference/config/analysis/ist0117/index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: DeploymentRequiresServiceAssociated
+layout: analysis-message
+owner: istio/wg-user-experience-maintainers
+test: no
+---
+
+当 Pods 资源没有与任何 Service 资源相关联时会出现该问题。
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/analysis/ist0117/index.md:8]: Pod

**Review by kebe7jun — APPROVED** [review]
LGTM

**Pull request merged**